pkg/management/component.go

```diff
@@ -294,6 +303,19 @@ func (c *component) CreateUser(ctx context.Context, realmName string, user api.U
 	reg := regexp.MustCompile(`[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}`)
 	userID := string(reg.Find([]byte(locationURL)))
 
+	// Store user in database
+	err = c.usersDBModule.StoreOrUpdateUser(ctx, realmName, dto.DBUser{
+		UserID:               &userID,
+		BirthLocation:        user.BirthLocation,
```

Comment:
If all fields are nil, do we need to create a row in DB?

Reply:
there is pro and cons about it .... I thought it would make sense to create it anyway but your comment make me think about it a second time and I think you are right.

Reply:
done